fxevent/console_test.go

```diff
@@ -160,18 +160,39 @@ func TestConsoleLogger(t *testing.T) {
 			give: &Provided{
 				ConstructorName: "bytes.NewBuffer()",
 				OutputTypeNames: []string{"*bytes.Buffer"},
+				Private:         false,
 			},
 			want: "[Fx] PROVIDE	*bytes.Buffer <= bytes.NewBuffer()\n",
 		},
+		{
+			name: "Provided privately",
+			give: &Provided{
+				ConstructorName: "bytes.NewBuffer()",
+				OutputTypeNames: []string{"*bytes.Buffer"},
+				Private:         true,
+			},
+			want: "[Fx] PRIVATE PROVIDE	*bytes.Buffer <= bytes.NewBuffer()\n",
```

Comment:
in terms of readability, something like PROVIDE(PRIVATE) might be easier to read since PROVIDEs are aligned among private/non-private.